src/project/qml/MuseScore/Project/internal/NewScore/TitleListView.qml

```diff
@@ -86,9 +86,23 @@ Item {
         width: parent.width
     }
 
+    StyledTextLabel {
+        id: noResultsFoundHint
+
+        anchors.centerIn: parent
+
+        font: ui.theme.bodyBoldFont
+
+        text: "No results found."
+
+        visible: view.numItems < 1
```

Comment:
You can use "count" directly here:

`visible: view.count < 1`

Therefore, numItems is not needed in this case